tests/AbstractProcessTest.php

```diff
@@ -333,4 +406,16 @@ private function getPhpBinary()
 
         return $runtime->getBinary();
     }
+    
+    private function getPhpCommandLine($phpCode)
+    {
+    	$cmd = $this->getPhpBinary() . ' -r ' . escapeshellarg($phpCode);
+    	
+    	if (defined('PHP_WINDOWS_VERSION_BUILD')) {
+    		// Windows madness! for some obscure reason, the whole command lines needs to be
+    		// wrapped in quotes (?!?)
+    		$cmd = '"'.$cmd.'"';
```

Comment:
Thanks! Tip by @auroraeosrose did the trick. Indeed, I can remove the quotes around the command by passing this parameter. I fixed the unit test to reflect that.